dev-docs/creating_bindings.md

```diff
@@ -0,0 +1,12 @@
+# Creating bindings for H3
+
+H3 is a C library, in part to make it simpler to create bindings for different programming languages. Each language usually has its own way to bind to C functions, but this document can serve as a starting point and list of tips.
```

Comment:
I might note the bindings that we're intending to release, so that no one wastes time implementing Java, JS, or Go.

Reply:
What about Python?

Reply:
Added, even Python. 😛

Reply:
Keep watching #21 :)